src/main/kotlin/se/uulm/snowballr/backend/repository/association/ProjectPaperTableRepo.kt

```diff
@@ -168,4 +180,20 @@ class ProjectPaperTableRepo(
                 it[createdBy] = userId
             }
     }
+
+    override suspend fun getProjectProgress(projectId: UUID): Float = db.query {
+        val allPapersCount = ProjectPaperTable.selectAll().where { ProjectPaperTable.projectId eq projectId }.count()
+        if (allPapersCount == 0L) {
+            0.0f
+        } else {
+            val fullyReviewedPapers = ProjectPaperTable.selectAll().where {
```

Comment:
I missed this the first time: Add the "Count" suffix here too to signal that it's a length of a list and not the list itself.

```suggestion
            val fullyReviewedPapersCount = ProjectPaperTable.selectAll().where {
```